pkg/api/login_oauth.go

```diff
@@ -97,8 +98,13 @@ func (hs *HTTPServer) OAuthLogin(ctx *models.ReqContext) {
 
 	code := ctx.Query("code")
 	if code == "" {
-		// FIXME: access_type is a Google OAuth2 specific thing, consider refactoring this and moving to google_oauth.go
-		opts := []oauth2.AuthCodeOption{oauth2.AccessTypeOffline}
+		opts := []oauth2.AuthCodeOption{oauth2.AccessTypeOnline}
```

Comment:
If access_type is s google specific thing should we really append it as default to all other providers?

Reply:
I left this because of backward compatibility and because I found other providers who implemented the same.

Reply:
At the end we decided to remove this because it is only applicable to Google and the default value is `online`. When the `accessTokenExpirationCheck` is enabled then the `access_type=offline` and `consent=prompt` are added to the query string.